aiinfra-cluster/modules/slurm-cluster/variables.tf

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+variable "project_id" {
+  description = "Name of the project to use for instantiating clusters."
+  type        = string
+}
+
+variable "deployment_name" {
+  description = "The deployment name. Default value is name_prefix-depl."
+  type        = string
+  default     = null
+}
+
+variable "zone" {
+  description = "GCP zone where to create resources. Only use if multi_zonal is false."
+  type        = string
+}
+
+variable "region" {
+  description = "GCP region where to create resources."
+  type        = string
+}
+
+variable "network_id" {
```

Comment:
I dont see the usage of this anywhere. Is this necessary ?

Reply:
Nope, this is a left over and I removed it.

Also kind of related to this: `network_self_link` vs `subnetwork_self_link`? Modules like the login node take both and require only one of them (their descriptions say `"Network/Subnet to deploy to. Either network_self_link or subnetwork_self_link must be specified"`). Which one should be provided?